picard/releasegroup.py

```diff
@@ -74,6 +77,10 @@ def _parse_versions(self, document):
 
             countries = countries_from_node(node)
 
+            if len(node['media']) > MAX_TRACKS:
+                tracks = "+".join([str(m['track-count']) for m in node['media'][:MAX_TRACKS]]) + '+…'
```

Comment:
How about:
```
tracks = "+".join([str(m['track-count']) for m in node['media'][:MAX_TRACKS]]) + '+… (' + _('Total discs') + len(node['media']) + ')'
```

Reply:
Unneeded. Media are already listed in "Format" field (ie. 100xCD)

Reply:
Ok.